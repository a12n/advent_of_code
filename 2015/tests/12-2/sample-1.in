../12-1/sample-1.in