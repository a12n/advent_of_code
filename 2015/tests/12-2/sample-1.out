../12-1/sample-1.out